SETUP.md

```diff
@@ -0,0 +1,81 @@
+# SETUP
+
+This guide is to get you setup with the tools you'll need for development.
+
+**Date**: May 24, 2002
+
+**Platform**: Fedora 35, 64-bit
+
+1. Install _SDKMan_, [link](https://sdkman.io/install).
+   1. `curl -s "https://get.sdkman.io" | bash`
+   2. `source "$HOME/.sdkman/bin/sdkman-init.sh"`
+   3. Verify the installation: `sdk version`
+2. Install _Coretto Java 8_, [link](https://sdkman.io/usage).
+   1. List the versions: `sdk list java`
+   2. Install the desired versions: `sdk install java 8.332.08.1-amzn`
+   3. If _JAVA_HOME_ is not set, re-source SDKMan's init script: `source "$HOME/.sdkman/bin/sdkman-init.sh"`
+3. Install the latest version of _Scala_: `sdk install scala`
+4. Install the latest version of _SBT_: `sdk install sbt`
+5. Install _Docker Engine_, _containerd_, and the _Docker Compose_ plugin.
```

Comment:
Docker isn't required for contribution. It's an optional to spin up a local cluster using kubernetes. There are other ways as well.

Reply:
I would mark this as optional -- and make it clear it's for running local clusters and integration tests, I think it should be included as a default but made clear that it's not strictly required for IDE development, only cluster level testing.

Reply:
Separated out Docker into its own section and clarified it as optional.